src/prefabs/structures/TimePicker/index.ts

```diff
@@ -0,0 +1,109 @@
+import { component, PrefabReference } from '@betty-blocks/component-sdk';
+import { updateOption } from '../../../utils';
+import { Configuration } from '../Configuration';
+import { options as defaults } from './options';
+
+export enum DateInputTypes {
```

Comment:
The input types are not necessary, because the type is determined by the file right?